controllers/contact-controller.js

```diff
@@ -170,6 +170,7 @@ function convertContactsToSearchResults(contacts) {
       const categories = 'TBD';
       const counselor = contact.twilioWorkerId;
       const notes = contact.rawJson.caseInformation.callSummary;
+      const { channel, conversationDuration } = contact;
```

Comment:
It's not clear to me why we are using this `{ }` notation to get the channel and conversationDuration, but not using it for the other properties of `contact`... see lines 165 and 166.  Let's be consistent, either way.

Reply:
If not using object destructuring notation I get an eslint error
![Screenshot from 2020-03-31 14-02-56](https://user-images.githubusercontent.com/15805319/78054525-99a0f400-7358-11ea-944d-9e508a00b8aa.png)
raised by this linting rule: `Use object destructuring eslint(prefer-destructuring)`
To avoid it I can
1. disable that eslint rule
2. do something like this
![Screenshot from 2020-03-31 14-07-41](https://user-images.githubusercontent.com/15805319/78054836-07e5b680-7359-11ea-9f79-c41db1f3a94d.png)

If destructuring is out, I'll rather go with 2. Is that ok?

Reply:
I believe your code is fine, @GPaoloni , you're right when you say:
> In the other properties, the variable name is different from the contact.property name (e.g. `const dateTime = contact.createdAt`), and thus the linting rule doesn't apply.

That explains why using destructuring in one place and not in the other. Also, this lint rule makes sure we're consistent, so I wouldn't change anything in this case.

Reply:
Oh... I see the issue.  I didn't realize that we were changing the variable name for contactId` and `dateTime`, so destructuring doesn't work.  Never mind.  I'm fine with your original solution.  I misread what was happening.